python/pfs/instmodel/makePfiDesign.py

```diff
@@ -1,50 +1,64 @@
 import numpy as np
-import pfs.instmodel.fieldDefinitions
+from pfs.instmodel.slit import Slit
+from pfs.instmodel.utils.generators import keepOdd, keepEven
+from pfs.instmodel.makePfsConfig import makeScienceDesign
 
 """Generate a PfiDesign"""
```

Comment:
General remark: following DAMD-42 `PfiDesign` is now `PfsDesign`. But this can be addressed in a separate ticket.